mock_log_client.go

```diff
@@ -4,9 +4,9 @@
 package trillian
 
 import (
-	"github.com/golang/mock/gomock"
-	"golang.org/x/net/context"
-	"google.golang.org/grpc"
+	gomock "github.com/golang/mock/gomock"
+	context "golang.org/x/net/context"
+	grpc "google.golang.org/grpc"
```

Comment:
Why have you renamed these imports?  The go readability review comments doc has a section on imports that states you should 'Avoid renaming imports except to avoid a name collision'.